.github/workflows/run_deploy_on_merge.yml

```diff
@@ -152,3 +152,23 @@ jobs:
             ${{ steps.matching_release.outputs.body }}
             ${{ env.COMMIT_MSG }}
 
+      # Update repository variables based on new release version and its MD5 hash
+      # These are used by the Conda package build Action
+      - uses: action-pack/set-variable@v1
+        with:
+          name: 'PHG2_VERSION'
+          value: '${{ env.VERSION }}'
+          token: ${{secrets.PHGV2CD}}
```

Comment:
I realize "secrets.<>" is used in existing code.  Where are the secrets values stored (ie what file is accessed?). Do we have these created for maizegenetics.net, or is this per project?

Reply:
Nevermind - I see from your slack posting this was generated for maize-genetics